scraper_data.py

```diff
@@ -33,5 +39,13 @@ def main(counties: Tuple[str,...], output:str) -> None:
     else:
         print(json.dumps(out,indent=2))
 
+    if not out: # all counties failed
+        return 70
+    elif failed_counties: # some counties failed
+        return 1
+    else: # success!
+        return 0
+
 if __name__ == '__main__':
-    main()
+    exit_code = main()
+    exit(exit_code)
```

Comment:
Ah, this doesn’t work for me, and I’m pretty sure it’s because Click commands usually don’t actually return their return value (see https://click.palletsprojects.com/en/7.x/commands/#command-return-values).

It’s probably better to just call `sys.exit(code)` up on lines 43 and 45.

Reply:
Makes sense - calling `sys.exit` on lines 43 and 45 instead to avoid getting caught up by click!